cli/common.go

```diff
@@ -131,16 +134,12 @@ func createRequest(slices []string) (controller.Request, error) {
 }
 
 var (
-	statusHeader = "Group | Units | FDState | FCState | SAState | IP | Machine"
-	statusBody   = "%s | %s | %s | %s | %s | %s | %s"
+	statusHeader = "Group | Units | FDState | FCState | SAState {{if .Verbose}}| Hash{{end}} | IP | Machine"
```

Comment:
I initially tried having two separate variables for the verbose case, but I felt that is ugly too. So I did the loop to go templates.